supplementary/devices/index.rst

```diff
@@ -471,6 +472,25 @@ Devices
      - GPS + GLONASS + (Beidou) Navigation **??**
      - Ambient light sensor, Proximity sensor, Motion G-sensor, Compass sensor, Gyro sensor, Magnetic sensor, Fingerprint sensor, Sensor Hub - **??**
      - Other **??**
+   * - :doc:`Xiaomi Redmi2/Prime <wt88047>`
```

Comment:
Without the wt88047 page existing, this will cause another warning and the build will fail.